main.nf

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl = 2
+
+/*
+    ~~~~~~~~~~~~~~~~~~~~~~~
+        Modules imports
+    ~~~~~~~~~~~~~~~~~~~~~~~
+*/
+include { AMRFINDER_PLUS } from './modules/amrfinder_plus'
+include { AMRFINDER_REPORT } from './modules/amrfinder_report'
+include { CRISPR_FINDER } from './modules/crisprcas'
+include { DIAMOND } from './modules/diamond'
+include { FASTA_WRITER } from './modules/fasta_writer'
+include { GBK_SPLITTER } from './modules/gbk_splitter'
+include { GENOMAD } from './modules/genomad'
+include { GFF_MAPPING } from './modules/gff_mapping'
+include { GFF_REDUCE } from './modules/gff_reduce'
+include { GFF_VALIDATOR } from './modules/validator'
+include { INTEGRONFINDER } from './modules/integronfinder'
+include { ISESCAN } from './modules/isescan'
+include { ICEFINDER } from './modules/icefinder'
+include { INTEGRATOR } from './modules/integrator'
+include { PROKKA } from './modules/prokka'
+include { RENAME } from './modules/rename_contigs'
+include { VIRIFY_QC } from './modules/virify_qc'
+
+/*
+    ~~~~~~~~~~~~~~~~~~~~
+        Help message    
+    ~~~~~~~~~~~~~~~~~~~~
+*/
+
+def helpMessage() {
+	log.info """
+	The Mobilome Annotation Pipeline is a wrapper that integrates the output of different tools designed for the prediction of plasmids, phages and autonomous integrative mobile genetic elements in prokaryotic genomes and metagenomes. The output is PROKKA gff file with extra entries for the mobilome. The defaul tools to run are ISEScan (insertion sequences), IntegronFinder (integrons), geNomad (virus and plasmids), ICEfinder (ICE and IME), PROKKA (cds prediction and general functional annotation), Diamond vs MobileOG database (mobilome functions), AMRfinderplus (AMR annotation), and CRISPRCas (crispr arrays). In addition, the user can provide PaliDIS results to be integrated with ISEScan and VIRify v2.0 results to integrate with geNomad annotation. If the user have a CDS prediction, it can be used to generate an extra gff file with the option --user_genes. See usage below:
```

Comment:
```suggestion
	The Mobilome Annotation Pipeline is a wrapper that integrates the output of different tools designed for the prediction of plasmids, phages and autonomous integrative mobile genetic elements in prokaryotic genomes and metagenomes. The output is PROKKA gff file with extra entries for the mobilome. The default tools to run are ISEScan (insertion sequences), IntegronFinder (integrons), geNomad (virus and plasmids), ICEfinder (ICE and IME), PROKKA (cds prediction and general functional annotation), Diamond vs MobileOG database (mobilome functions), AMRfinderplus (AMR annotation), and CRISPRCas (crispr arrays). In addition, the user can provide PaliDIS results to be integrated with ISEScan and VIRify v2.0 results to integrate with geNomad annotation. If the user have a CDS prediction, it can be used to generate an extra gff file with the option --user_genes. See usage below:
```